### ISSUE TITLE
Ctrl + A in file browser listing should select all files

### ISSUE THREAD

**Issue by krassowski** [jupyterlab/jupyterlab#16690]
### Problem

Moving or downloading multiple files is cumbersome as I need to select all by clicking multiple files, using shift + click or using the checkbox. It should be easier. 

### Proposed Solution

All UI Linux file browsers I know support Ctrl + A shortcut for selecting all. I think this is also the case for Windows. Not sure what shortcut is used on Mac.

### Additional context

It should be tested that the shortcut is not active when the file name is edited as in that case Ctrl + A should select everything in the edit field.

We would want to add a new command, say `filebrowser:select-all` and then a shortcut here:

https://github.com/jupyterlab/jupyterlab/blob/35bab6e06d8e4f347219f91a4f0315a4169dfe09/packages/filebrowser-extension/schema/browser.json#L143-L184

**Comment by JasonWeill** [comment]
<kbd>CMD</kbd>+<kbd>A</kbd> does the same thing in Finder, the macOS file browser.

**Comment by JasonWeill** [comment]
From 2024-09-04 frontends meeting: if all files are selected, and the user presses <kbd>ENTER</kbd> or <kbd>RETURN</kbd>, should we display a warning dialog asking the user whether they really want to open a lot of files at once? This is particularly important in notebook or in JupyterLab in single-document mode, where every file opens in its own tab/window, so closing a large number of windows or tabs may be cumbersome. This might merit a separate issue.
